### PR TITLE
Added html for section 1

### LINKED ISSUE #2: html section 1
We need to create the layout for the section 1

### PR DESCRIPTION
**Pull request by MaksimFrontend**

### CODE REVIEW

File: assets/.gitignore
Comment (Evbgsl) [inline assets/.gitignore:1]: Move this file to the root folder

File: assets/README.MD
Comment (Evbgsl) [inline assets/README.MD:1]: Move this file to the root folder

File: index.html [hunk 1]
```diff
@@ -3,13 +3,31 @@
...
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Peaceful shopping mindful money</h1>
+          <p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/apple.svg" alt="Apple icon">
+            </a>
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/google.svg" alt="Google icon">
```
Comment (Evbgsl) [inline index.html]: Use / for the single tags like this <..... />

File: index.html [hunk 2]
```diff
@@ -3,13 +3,31 @@
...
   <body>
     <!-- Шапка сайта -->
     <header>Шапка сайта</header>
     <!-- Секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Peaceful shopping mindful money</h1>
+          <p class="copy">See how we can help with making your shopping experience and money management more ZEN.</p>
```
Comment (Evbgsl) [inline index.html]: Use this format

```
<p class="copy">
See how we can help with making your shopping experience 
and money management more ZEN.
</p>
```

File: index.html [hunk 3]
```diff
@@ -3,13 +3,31 @@
...
+          <div class="links">
+            <a class="link" href="https://apple.com">
+              <img src="./assets/icons/apple.svg" alt="Apple icon">
```
Comment (Evbgsl) [inline index.html]: Use / for the single tags like this <..... />

File: index.html [hunk 4]
```diff
@@ -3,13 +3,31 @@
...
+            </a>
+            <a class="link" href="https://google.com">
+              <img src="./assets/icons/google.svg" alt="Google icon">
+            </a>
+          </div>
+        </div>
+        <div class="image-block">
+          <img src="./assets/images/1.jpg" alt="Bag and device">
```
Comment (Evbgsl) [inline index.html]: Use / for the single tags like this <..... />

File: index.html [hunk 5]
```diff
@@ -3,13 +3,33 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Evbgsl) [inline index.html]: Use / for close single tag

File: index.html [hunk 6]
```diff
@@ -3,13 +3,33 @@
...
     <!-- Шапка сайта -->
     <header>Шапка сайта</header>
     <!-- Секция 1 -->
-    <section>Секция 1</section>
+    <section class="section" id="warranty">
+      <div class="wrapper">
+        <div class="text-block">
+          <h1 class="title">Peaceful shopping mindful money</h1>
+          <p class="copy">
+            See how we can help with making your shopping experience and money management more ZEN.
```
Comment (Evbgsl) [inline index.html]: try splitting the line into several lines by pressing Enter

**Review by Evbgsl — COMMENTED** [review]
- fix the number of indents in the VS code settings. There should be 2 pieces instead of 4.
- use auto-formatting shift+alt+F (be careful - empty lines may be added - they must be removed)

**Review by Evbgsl — APPROVED** [review]

**Review by KoloskovaAlla — APPROVED** [review]

**Pull request merged**